src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -204,7 +233,9 @@ public virtual void OnEvaluationStarted(EventArgs e)
         public virtual void OnEvaluationCompleted(EvaluationCompletedEventArgs e)
         {
             this.HasRunWithoutCrash = e.EvaluationSucceeded;
-            
+            GraphRunInProgress = false;
+            EvaluationCount++;
```

Comment:
Sometimes `OnEvaluationCompleted` is called even when evaluation is skipped for various reasons. Do you want this counter to increment even for these cases?

Reply:
I guess I am not sure, I do not have any good reasons to go one way or the other
I guess it is safer to leave the counter as it was...